src/main/groovy/org/mockito/release/internal/comparison/DownloadPreviousReleaseArtifactsTask.java

```diff
@@ -0,0 +1,82 @@
+package org.mockito.release.internal.comparison;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.GradleException;
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.gradle.api.tasks.*;
+import org.mockito.release.notes.util.IOUtil;
+
+import java.io.File;
+
+/**
+ * Downloads artifacts from last release and stores them to local files for further comparison
+ * Currently it downloads .pom and -sources.jar
+ */
+public class DownloadPreviousReleaseArtifactsTask extends DefaultTask {
+
+    private static final Logger LOG = Logging.getLogger(DownloadPreviousReleaseArtifactsTask.class);
+
+    @Input
+    private String previousVersionPomUrl;
+    @Input
+    private String previousVersionSourcesJarUrl;
+
+    @OutputFile
+    private File previousVersionPomLocalFile;
+    @OutputFile
+    private File previousVersionSourcesJarLocalFile;
+
+
+    @TaskAction public void downloadPreviousReleases(){
+        if(previousVersionPomUrl == null || previousVersionSourcesJarUrl == null){
+            throw new GradleException("You have to configure previousVersionPomUrl and previousVersionSourcesJarUrl to use DownloadPreviousReleaseArtifactsTask.\n"
+                    + "If you use one of the supported publishing plugins default url will be configured for you.\n"
+                    + "Currently supported plugins: Bintray"
+            );
+        }
+        downloadRemoteFile(".pom", previousVersionPomUrl, previousVersionPomLocalFile);
+        downloadRemoteFile("-sources.jar", previousVersionSourcesJarUrl, previousVersionSourcesJarLocalFile);
+    }
+
+    private void downloadRemoteFile(String extension, String remoteUrl, File localFile) {
+        LOG.lifecycle("Downloading remote artifact\n" +
```

Comment:
I like this message. When we download stuff or start potentially long operations it is always useful to give a heads-up to the user.